tools/validators/instance_validator/validate/handler.py

```diff
@@ -178,12 +195,21 @@ def _ParseArgs(self, args):
         help='Timeout duration (in seconds) for telemetry validation test',
         metavar='timeout')
 
+    parser.add_argument(
+        '-r', '--report-filename',
+        dest = 'report_filename',
+        required = False,
+        default = None,
+        help = 'Filename for the validation report',
+        metavar = 'report-filename')
```

Comment:
can you please update the readme file of the instance validator to mention these args.

I would propose to guide the user as well where they need first to validate without the telemetry args and then once it passes then they should add those args.

Reply:
Done.